mdp/hinet/layer_online.py

```diff
@@ -0,0 +1,149 @@
+"""
+Module for OnlineLayers.
```

Comment:
I wonder if it would be better to put all the online stuff into a new subpackage (like hinet and parallel)?
This is just a thought, I haven't really thought it through yet.